src/hooks/useScreenWidthFromThreshold.ts

```diff
@@ -7,6 +7,7 @@ export const useScreenWidthFromThreshold = (threshold: number): { lessThanThresh
 
   useEffect(() => {
     const listener = () => {
+      const media = window.matchMedia(`(max-width: ${threshold}px)`);
```

Comment:
remove _media_ from line 5 since it's redundant

Reply:
removed